doc/TEMPLATES.md

```diff
@@ -286,9 +284,6 @@ const fragment = template.content.cloneNode(true);
 
 html`<div>${fragment}</div>`;
 // <div><script>console.log('hello world');</script></div>
```

Comment:
Can we update this example to avoid this antipattern of inline script? IMO this is not the use case we want to exemplify

Reply:
After our discussion: svg makes more sense because it has children nodies